Tests/TestingApps/PaywallsTester/PaywallsTester/Config/SamplePaywalls.swift

```diff
@@ -9,7 +9,7 @@
 
 import Foundation
 import RevenueCat
-@testable import RevenueCatUI
```

Comment:
Hmm I guess if we're adding/removing the `public` on every build this won't be needed yeah... And it would break uploads to app store connect I imagine.

Reply:
yes, that's what was breaking the upload